src/main/resources/templates/fail.html

```
@@ -0,0 +1,55 @@
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="https://static.toss.im/icons/png/4x/icon-toss-logo.png" />
+    <link rel="stylesheet" type="text/css" href="/css/style.css" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>토스페이먼츠 샘플 프로젝트</title>
+  </head>
+
+  <body>
+    <div id="info" class="box_section" style="width: 600px">
+      <img width="100px" src="https://static.toss.im/lotties/error-spot-no-loop-space-apng.png" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)